index.html

```diff
@@ -38,12 +47,8 @@
                 <button class="add_folder_btn" id="addFolderBtn">폴더 추가 +</button>
                 <div id="addFolderModal" class="modal">
                     <div class="modal_container">
-                        <div class="close">
-                            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-                                <circle cx="12" cy="12" r="12" fill="#E7EFFB"/>
-                                <path d="M8 8L16 16" stroke="#9FA6B2" stroke-width="2" stroke-linecap="round"/>
-                                <path d="M16 8L8 16" stroke="#9FA6B2" stroke-width="2" stroke-linecap="round"/>
-                            </svg>                                
+                        <div class="close" id="addFolderCloseBtn">
```

Comment:
이 부분 실제로 하는 일이 button 의 역할이라 button 태그 써주시는 게 더 시맨틱할 거 같아요!
```suggestion
                        <button class="close" id="addFolderCloseBtn">
```